KinSDK/KinSDK/source/Watchers.swift

```diff
@@ -51,16 +51,23 @@ public class BalanceWatch {
                     for op in opsMeta {
                         for change in op.changes {
                             switch change {
-                            case .LEDGER_ENTRY_CREATED: break
-                            case .LEDGER_ENTRY_REMOVED: break
-                            case .LEDGER_ENTRY_UPDATED(let le):
+                            case .LEDGER_ENTRY_CREATED(let le),
+                                 .LEDGER_ENTRY_UPDATED(let le):
```

Comment:
@avi-kik turns out the account condition is needed in the update case when funding the account after creation.